src/edu/psu/sweng500/team8/gui/GridPanel.java

```diff
@@ -305,6 +309,10 @@ private void markGivenCell(Cell cell) {
 		HighlightPainter painter = new DefaultHighlighter.DefaultHighlightPainter(
 				Color.green);
 
+		/* TODO - THIS MAY NOT BE THE BEST PLACE TO HAVE THIS SET BUT I NEED THIS TO MAKE PENCIL MARK WORKING
+		 * DO NOT FORGET REVISIT AND UPDATE ValueType logic */
+		cell.setType(ValueType.Given);
```

Comment:
This shouldn't be necessary. The type is set when the puzzle is constructed before markGivenCell is called